pkg/bpf/detect_linux.go

```diff
@@ -508,14 +510,38 @@ func HasUprobeRefCtrOffset() bool {
 	return uprobeRefCtrOffset.detected
 }
 
+func detectAuditLoginuid() bool {
+	enabled, err := kernels.DetectConfig(kernels.CONFIG_AUDIT)
+	if err != nil {
+		if _, err := os.Stat("/proc/self/loginuid"); err != nil {
+			logger.GetLogger().Error("failed to stat /proc/self/loginuid", logfields.Error, err)
```

Comment:
nit: maybe remove the log here or if the error is relevant: return an error and log it at the very last step. Or if it's not important, we can just return false here. It should be fine in this case.